.gitignore

```diff
@@ -1,5 +1,6 @@
 /dist
 /autogen/genstatic/gen.go
+/autogen/gentemplates/gen.go
```

Comment:
Not sure this is right.

Reply:
I think the file `/autogen/gentemplates/gen.go` should be generate when build, likes `/autogen/gen.go` in **v1.4**.